plugins/techdocs-react/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@backstage/plugin-techdocs-react",
+  "description": "Shared frontend utilities for TechDocs and Addons",
+  "version": "0.0.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "web-library"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/techdocs-react"
+  },
+  "keywords": [
+    "backstage",
+    "techdocs"
+  ],
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "backstage-cli package build",
```

Comment:
This need to include `--experimental-type-build` for the alpha types to be built